nfl_data_py/tests/nfl_test.py

```diff
@@ -8,12 +8,22 @@ def test_is_df_with_data(self):
         s = nfl.import_pbp_data([2020])
         self.assertEqual(True, isinstance(s, pd.DataFrame))
         self.assertTrue(len(s) > 0)
+
+    def test_is_df_with_data_thread_requests(self):
+        s = nfl.import_pbp_data([2020, 2021], thread_requests=True)
+        self.assertEqual(True, isinstance(s, pd.DataFrame))
+        self.assertTrue(len(s) > 0)
 		
```

Comment:
Out of scope for this PR, but it reminded me that we should have a test for the caching behavior.
```suggestion
		
    # TODO: Add test(s) for cache=True
    
```